smdebug/tensorflow/keras.py

```diff
@@ -497,6 +498,10 @@ def save_smdebug_logs(self, logs):
                         else set()
                     )
                 for t_name, t_value in tensors_to_save:
+                    if isinstance(t_value, dict):
```

Comment:
can it get to this point in non-eager mode?

Reply:
Both eager and non-eager modes execute this code-block.